contentcuration/contentcuration/urls.py

```diff
@@ -122,18 +122,9 @@ class ContentKindViewSet(viewsets.ModelViewSet):
 
 
 class ContentNodeViewSet(BulkModelViewSet):
-    queryset = ContentNode.objects.all()
+    queryset = ContentNode.objects.none()
```

Comment:
I think this might cause issues because the viewset uses the queryset as the base for all filtering and getting, so this will always return no result?

Better to override the `list` method and reject requests to the list endpoint that don't apply any filters.